src/inverseRobot.js

```diff
@@ -24,7 +24,16 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
-}
+  const fixedRobot = {};
+
+  for (const key in robot) {
+    if (robot[key] in fixedRobot) {
+      return null;
+    }
```

Comment:
Better use `hasOwnProperty` to check is property in object. because `in` will also return true if key gets found somewhere in the prototype chain, otherwise Object.hasOwnProperty (like the name already tells us), will only return true if key is available on that object directly (its "owns" the property).